tests/unit/core/TestInstallAddon.js

```diff
@@ -1322,7 +1385,7 @@ describe(`${__filename}: withInstallHelpers`, () => {
       return uninstall(addon).then(() => {
         sinon.assert.calledWith(fakeTracking.sendEvent, {
           action: TRACKING_TYPE_THEME,
```

Comment:
`uninstall` calls `getAction` for this value, so you should do that in the test as well.